integration/deb/install_deb.go

```diff
@@ -0,0 +1,33 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os/exec"
+	"strconv"
+	"time"
+)
+
+const (
+	numberRetry = 5
+	waitTime    = time.Minute
+)
+
+// Try to install amazon cloud watch agent deb. If you can't install wait 1 minute
+func main() {
+	for i := 0; i < numberRetry; i++ {
+		log.Print("Attempt to install cloud watch agent deb number " + strconv.Itoa(i+1))
+		out, err := exec.Command("bash", "-c", "sudo apt install -y ./amazon-cloudwatch-agent.deb").Output()
+		if err != nil {
+			log.Print(fmt.Sprint(err) + string(out))
+		} else {
+			log.Print("Agent Installed")
+			return
+		}
```

Comment:
nit: we could rearrange the logic here so that instead of `if err != nil`, we just break early and remove the `else`:
```go
if err == nil {
    log.Println("agent installed")
   return
}

time.Sleep(waitTime)
```